backend/__fixtures__/serviceaccounts.js

```diff
@@ -14,8 +14,10 @@ const { getTokenPayload } = require('./auth')
 
 const serviceAccountList = [
   getServiceAccount({ namespace: 'garden-foo', name: 'robot', createdBy: 'foo@example.org' }),
+  getServiceAccount({ namespace: 'garden-foo', name: 'robot-two-secrets', createdBy: 'foo@example.org' }),
```

Comment:
I think it is better to not test these cases `missing-secret` and `two-secretes` in acceptance tests. We should rather test these cases in a unit test of the member manager.

Reply:
moved to unit tests